gulp/clean.js

```diff
@@ -10,7 +10,8 @@ module.exports = function (gulp, gutil) {
   });
 
   gulp.task('clean-dist', function () {
-    return del()('dist', { force: true });
```

Comment:
@adamayres @keerthana-ceg Could we have defined a separate task for 'clean-plugin', alongside 'clean-dist', with the used gutil.env.newStructure condition?

Reply:
Yes, but then you'd need some way to switch between which clean call is used by which environment. Currently they both use `clean` which proxies to this task.